backend/routes/rankings.js

```diff
@@ -1,52 +1,81 @@
 import express from "express";
 import { Follow } from "../models/follow.js";
 import { Stock } from "../models/stock.js";
+import { Like } from "../models/like.js";
+import { Dislike } from "../models/dislike.js";
+import { compareStocksByPoints } from "../utils.js";
+
+const FOLLOW_POINTS = 25;
+const LIKE_POINTS = 20;
+const DISLIKE_POINTS = -20;
```

Comment:
How did you decide to do these values (vs. smaller numbers like 5, 1, -1 or something else)?

Reply:
Not sure actually, I just kind of played with the first numbers that came to my head.